src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -8,6 +8,7 @@ tags:
   - name: Groups
   - name: Issue Trackers
   - name: Person
+  - name: Published Binaries
```

Comment:
Binaries are not technically always correct. Historically we use it, however today a better term exists: Artifacts or Job Artifacs. Artifacts refer to the output of your build process that you would deploy/install. This could be a repo file, tar or binary. You then deploy these artifacts onto your system. Example of non binary artifact: `osc api '/published/zypp:SLE-15-Branch/SLE_15_Update/zypp:SLE-15-Branch.repo'`

Reply:
I'm with you, `binaries` is not the best name for that heterogeneous output. However, I believe that by using `Artifacts` in the new API documentation, the users can struggle when trying to find the endpoint or understanding what we are referring to. Moreover, they won't be able to match `Artifacts` with any element in the UI, where we still use `Binaries`.

I really like the idea, so I propose to track it for future refactoring or another version of the API.